cmd/s3-peer-client.go

```diff
@@ -19,125 +19,74 @@ package cmd
 import (
 	"encoding/json"
 	"fmt"
-	"net/rpc"
 	"net/url"
 	"path"
 	"sync"
-	"time"
 )
 
 type s3Peers struct {
-	// A map of peer server address (in `host:port` format) to RPC
-	// client connections.
-	rpcClients map[string]*AuthRPCClient
+	// A map of peer server address (in `host:port` format) to
+	// BucketMetaState interface
+	bmsClients map[string]BucketMetaState
 
-	mutex *sync.RWMutex
-
-	// Is single-node?
-	isDistXL bool
+	// Is distributed?
+	IsDistXL bool
 
 	// Slice of all peer addresses (in `host:port` format).
-	peers []string
+	Peers []string
```

Comment:
Peers includes 'itself' right? In that sense the name 'peers' may be a bit misleading